docs/topics/db/examples/proxy_models.txt

```diff
@@ -0,0 +1,96 @@
+===========================
+Proxy model as relationship
+===========================
```

Comment:
I like the idea of this page, but would like to see a few more words covering the utility. Also I'm not sure there's extra value from maintaining examples for all three kinds of relationships - if we show it with just `ForeignKey` readers should be able to extend it to `ManyToManyField` and `OneToOneField` pretty naturally.

Reply:
Oh, and we should also have a link to the page from the proxy models section that you've also touched - perhaps a "For more examples using proxy models, see *Proxy model examples*"